x-pack/legacy/plugins/reporting/export_types/printable_pdf/server/lib/generate_pdf.js

```diff
@@ -27,27 +27,35 @@ const formatDate = (date, timezone) => {
   return moment.tz(date, timezone).format('llll');
 };
 
-
 function generatePdfObservableFn(server) {
   const screenshotsObservable = screenshotsObservableFactory(server);
   const captureConcurrency = 1;
 
   const urlScreenshotsObservable = (urls, conditionalHeaders, layout, browserTimezone) => {
     return Rx.from(urls).pipe(
-      mergeMap(url => screenshotsObservable(url, conditionalHeaders, layout, browserTimezone),  // eslint-disable-line no-unused-vars
-        (outer, inner) => inner,
+      mergeMap(url => screenshotsObservable(url, conditionalHeaders, layout, browserTimezone),
```

Comment:
`(outer, inner) => inner,` being removed is from https://github.com/elastic/kibana/pull/40284/files#r305170223